extract_unique_sensors_01.py

```diff
@@ -1,6 +1,14 @@
+import argparse
 from urllib.request import urlopen
 from pathlib import Path
 
+parser = argparse.ArgumentParser(description='Extract unique sensors')
+parser.add_argument('--file_name', help='Actual name of the file')
+parser.add_argument('--url', help='Location of SensorList.txt')
```

Comment:
The description of the argument is a bit confusing.
From [Wikipedia](https://en.wikipedia.org/wiki/URL), 
> A Uniform Resource Locator (URL), colloquially termed a web address,[1] is a reference to a web resource that specifies its location on a computer network and a mechanism for retrieving it. A URL is a specific type of Uniform Resource Identifier (URI),[2][3] although many people use the two terms interchangeably

Reply:
@mamkhari